README.md

```diff
@@ -59,6 +63,18 @@ You may generate a semantic model for a given list of fully qualified tables fol
 
 All generated semantic models by default are saved either under `semantic_model_generator/output_models` if running from the root of this project or the current directory you're in.
 
+### Generation and validation using admin app
```

Comment:
add this under workflow section?

https://github.com/Snowflake-Labs/semantic-model-generator/pull/51/files/780c0d778644f23224350017e42ca869a5ae91c9..2f0a8cad4a512266c1d27b46497d3b8aae69ab86#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5